front/app/components/SeatInfo/index.tsx

```diff
@@ -13,52 +13,61 @@ import {
 // Hooks
 import useAppConfiguration from 'api/app_configuration/useAppConfiguration';
 import useSeats from 'api/seats/useSeats';
+import { TSeatType } from 'api/seats/types';
+import { TSeatNumber } from 'api/app_configuration/types';
 
 // Intl
 import messages from './messages';
-import { FormattedMessage, useIntl } from 'utils/cl-intl';
+import { FormattedMessage, MessageDescriptor, useIntl } from 'utils/cl-intl';
 
 // Utils
 import { isNil } from 'utils/helperUtils';
 
-type SeatInfoType = {
-  seatType: 'collaborator' | 'admin';
+type Props = {
```

Comment:
Any reason we are generalising here? I think the more specific the better 🤔

Reply:
We always call props type Props, right? 🙂 Search the codebase for `interface Props` (or less often used `type Props`).

Reply:
That I have seen and didn't like. Is there a good reason for that? I think the specific one is better but I have no strong opinion

Reply:
I don't know. It makes sense to call props props I'd say. This is how we do it in I think 99% of components. I've never had a problem with it. It makes it clear that it's the defintion of props types. SeatInfoType on the other would not make it clear that this is about props, rather a general type definition. I'd then have to check where it's used to see that it's used for props type definitions.

Reply:
Like I said, I have no strong opinions but general names tend to make looking for things harder. If it is about props, it could be named `SeatInfoProps`

Reply:
Happy to leave as is anyway

Reply:
Yes, SeatInfoProps is definitely fine for me 👍